build.yaml

```diff
@@ -876,16 +876,14 @@ common_cli:
     tar --verbose --list --file "${BOX_NAME}" || abort_build
 
   run_lint: &run_lint |
-    # we need to unset PIP_INDEX_URL since it causes tox to fail.
-    # in order to use our pip caching we need to append --trusted-hosts
-    # which we currently can't do through tox.
-    unset PIP_INDEX_URL
+    # Disable exiting on error, so we can run multiple lints
+    set +e
     # run flake8 lint tests on Flocker source code
-    tox -e lint
-
-  install_flake8: &install_flake8 |
-    # flake8 is used by lint tests on Flocker source code
-    pip install flake8 ${PIP_ADDITIONAL_OPTIONS}
+    flake8 --format=pylint --output flake8.lint flocker
```

Comment:
I think tox ignores a couple of files. This should too I guess. It does reveal that the two definitions will be a pain to keep in sync. Could you perhaps rewrite the tox commands to do these things and to write the files using tee?